Section on PI's red LED in Troubleshooting (#444)
pupil-labs/pupil-docs: src/invisible/user-guide/troubleshooting.md

   cable rather than another 3rd party cable. Such cables are available
   in a wide variety of quality and using cables of insufficient quality
   can lead to loss of connection.

**A red LED is blinking on my Pupil Invisible glasses!**

> The LED is indicating that there is a problem that may affect further data collection and tries to grab your attention. To get details on the problem, open the Pupil Invisible Companion app, which will show an error description.

Reviewer: ```suggestion
The LED blinks to try to get the wearer's attention. The red blinking LED indicates that there is a problem that may affect further recording. To get details, open the Pupil Invisible Companion app to view an error description.
```

> The LED is indicating that there is a problem that may affect further data collection and tries to grab your attention. To get details on what the problem exactly is open the Pupil Invisible Companion app, which will show an error description.

Reviewer 2: ```suggestion
The LED is indicating that there is a problem that may affect further data collection and tries to grab your attention. To get details on what the problem exactly is, open the Pupil Invisible Companion app, which will show an error description.
```

> The potential problems incude:

Reviewer: ```suggestion
The potential problems include:
```

> - Low battery on the Companion device.

Reviewer: ```suggestion
- Low battery on the Companion Device.
```

> - Little storage left on the Companion device.

Reviewer: ```suggestion
- Low storage space remaining on the Companion Device.
```

> - Unstable connection to the Pupil Invisible device during a recording.

Reviewer: ```suggestion
- Unstable connection to the Pupil Invisible Glasses device during a recording.
```
